.github/workflows/ci.yaml

```diff
@@ -0,0 +1,110 @@
+name: CI
```

Comment:
There is a file for CI named `build.yml`, why you add new file for CI?

Reply:
@Kaaveh I just used it for the convention